app/components/fastboot-only/body-bottom.js

```diff
@@ -38,8 +38,8 @@ export default Component.extend({
       'language',
       'qualarooUrl',
       'isTestWiki',
+      'scriptPath',
     );
-    wikiVariables.scriptPath = this.wikiVariables.articlePath.replace(/\/$/, '');
```

Comment:
For some reason, this fix brakes incrementing pv_number on articles in a language different than 'en'. However, if it causes problems on production it can be merged and we will work on a fix ASAP.

Reply:
Live QA confirmed that the way of testing was wrong. Fix works.